app/components/myissues.jsx

```diff
@@ -0,0 +1,161 @@
+var React = require("react");
+var getJSON = require("./getJSON.jsx");
+var { auth, AuthMixin } = require("./auth.jsx");
+var APIServer = "/api";
+var Labels = require("./labels.jsx");
+
+var IssueCard = React.createClass({
+  getInitialState: function() {
+    return {
+    };
+  },
+
+  getShortRepoString: function (fullUrl) {
+    var s = fullUrl;
+    s = s.replace("https://github.com/", "");
+    var i = s.indexOf("/");
+    s = s.slice(i+1);
+    i = s.indexOf("/");
+    s = s.slice(0, i);
+    return s;
+  },
+
+  render: function() {
+    var data = this.props.data;
+    var repo = this.getShortRepoString(data.html_url);
```

Comment:
ah nice, that's missing from [the github docs](https://developer.github.com/v3/issues/)